src/Backend/Modules/Blog/Domain/CommentRepository.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Backend\Modules\Blog\Domain;
+
+use Common\Locale;
+use Doctrine\DBAL\Connection;
+use Doctrine\ORM\EntityRepository;
+
+class CommentRepository extends EntityRepository
```

Comment:
the file structure is wrong.
Should be in a directory Comment

Reply:
fixed in 3cfb811e0b4fbd409e62096e036aa025efc55f88